src/styles/auro-tail.scss

```diff
@@ -0,0 +1,114 @@
+@use '@aurodesignsystem/webcorestylesheets/dist/bundled/type/classes.alaska.min.css';
+@use './tokens.scss';
+
+.tail {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  vertical-align: middle;
+  min-width: var(--container-width);
+  min-height: var(--container-width);
+}
+
+/* hyperlink styling */
+.tail > .tail-hyperlink {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-decoration: none;
+  min-width: var(--container-width);
+  min-height: var(--container-width);
+}
+
+.tail > .tail-hyperlink .label {
+  display: block;
+  margin-top: var(--href-label-gap);
+  text-align: center;
+  min-width: var(--container-width);
+  white-space: nowrap;
+  overflow: visible;
+  text-overflow: clip;
+}
+
+.border {
+  position: relative;
+  width: var(--container-width);
+  height: var(--container-width);
+  border-radius: var(--border-radius-full);
+  display: grid;
+  place-items: center;
+  /* Center the tail when featured in a hyperlink */
+  margin: 0 auto;
+  box-sizing: content-box;
+}
+
+/* For tails with borders */
+.border.has-border {
+  border: var(--border-width, var(--border-width-default)) solid var(--border-color, var(--border-color-default));
```

Comment:
Typically we set the colors in a `color.scss` file. Not completely necessary if there's only a couple colors, but something to keep in mind.